Lockscreen PIN reset limitations (#9023)
MicrosoftDocs/windows-itpro-docs: windows/security/identity-protection/hello-for-business/hello-feature-pin-reset.md

 3. Follow the instructions provided by the provisioning process
 4. When finished, unlock your desktop using your newly created PIN.

> you may find PIN reset only works from settings post login and This lock screen PIN reset will not work if you have any matching limitation of SSPR password reset from lock screen. refer the below doc
> https://docs.microsoft.com/en-us/azure/active-directory/authentication/howto-sspr-windows#general-limitations

Reviewer: I believe we should use correct sentence casing (starting sentences with an uppercase letter) as well as making sure the phrasing is consistent with other docs.
```suggestion
You may find that PIN reset only works from the settings post login, and "This lock screen PIN reset" will not work if you have any matching limitation of SSPR password reset from lock screen. For more information, please read the [Enable Azure Active Directory self-service password reset at the Windows sign-in screen - **General limitations**](https://docs.microsoft.com/en-us/azure/active-directory/authentication/howto-sspr-windows#general-limitations)
```
Please also note that I do not know how the sentence part "This lock screen PIN reset" fits with the rest of the sentence or the document on the whole. (Further copy review suggestions and improvements are more than welcome.)

Author: I think, you can remove word "This".. and it can be like below
 "lock screen PIN reset" will not work if you have any matching limitation of SSPR password reset from lock screen.

Author: corrected and submitted updated PR now. Please review

> >[!NOTE]
> > Visit the [Windows Hello for Business Videos](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-videos.md) page and watch the [Windows Hello for Business forgotten PIN user experience](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-videos#windows-hello-for-business-forgotten-pin-user-experience) video.

Reviewer: For consistency, please add the following added MarkDown indent marker compatibility spacing:
```suggestion
> [!NOTE]
> Visit the [Windows Hello for Business Videos](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-videos.md) page and watch the [Windows Hello for Business forgotten PIN user experience](https://docs.microsoft.com/windows/security/identity-protection/hello-for-business/hello-videos#windows-hello-for-business-forgotten-pin-user-experience) video.
```

 3. Follow the instructions provided by the provisioning process
 4. When finished, unlock your desktop using your newly created PIN.

> you may find PIN reset only works from settings post login and "lock screen" PIN reset function will not work if you have any matching limitation of SSPR password reset from lock screen. refer the below doc
> https://docs.microsoft.com/en-us/azure/active-directory/authentication/howto-sspr-windows#general-limitations

Reviewer: ```suggestion
You may find that PIN reset from settings only works post login, and that the "lock screen" PIN reset function will not work if you have any matching limitation of SSPR password reset from lock screen. For more information, see [Enable Azure Active Directory self-service password reset at the Windows sign-in screen - **General limitations**](https://docs.microsoft.com/en-us/azure/active-directory/authentication/howto-sspr-windows#general-limitations).
```

 3. Follow the instructions provided by the provisioning process
 4. When finished, unlock your desktop using your newly created PIN.

> You may find that PIN reset from settings only works post login, and that the "lock screen" PIN reset function will not work if you have any matching limitation of SSPR password reset from the lock screen. For more information, see [Enable Azure Active Directory self-service password reset at the Windows sign-in screen - **General limitations**](https://docs.microsoft.com/en-us/azure/active-directory/authentication/howto-sspr-windows#general-limitations).

Reviewer 2: ```suggestion
You may find that PIN reset from settings only works post login, and that the "lock screen" PIN reset function will not work if you have any matching limitation of SSPR password reset from the lock screen. For more information, see [Enable Azure Active Directory self-service password reset at the Windows sign-in screen - **General limitations**](https://docs.microsoft.com/azure/active-directory/authentication/howto-sspr-windows#general-limitations).
```

 3. Follow the instructions provided by the provisioning process
 4. When finished, unlock your desktop using your newly created PIN.

> You may find that PIN reset from settings only works post login, and that the "lock screen" PIN reset function will not work if you have any matching limitation of SSPR password reset from lock screen. For more information, see [Enable Azure Active Directory self-service password reset at the Windows sign-in screen - **General limitations**](https://docs.microsoft.com/en-us/azure/active-directory/authentication/howto-sspr-windows#general-limitations).

Reviewer: Sorry, I forgot a minor detail in this sentence:
```suggestion
You may find that PIN reset from settings only works post login, and that the "lock screen" PIN reset function will not work if you have any matching limitation of SSPR password reset from the lock screen. For more information, see [Enable Azure Active Directory self-service password reset at the Windows sign-in screen - **General limitations**](https://docs.microsoft.com/en-us/azure/active-directory/authentication/howto-sspr-windows#general-limitations).
```
